instantsearch/src/commonMain/kotlin/com/algolia/instantsearch/searcher/hits/internal/DefaultHitsSearcher.kt

```diff
@@ -96,10 +123,23 @@ internal class DefaultHitsSearcher(
     }
 
     private fun onSingleQuerySearchResponse(responses: List<ResponseSearch>) {
-        response.value = responses.firstOrNull()
+        val responseSearch = responses.firstOrNull()
+        response.value = responseSearch
+        if (responseSearch != null && isAutoSendingHitsViewEvents) {
+            val events = getViewEvents(responseSearch)
+            if (events.isNotEmpty()) {
+                coroutineScope.launch(exceptionHandler) {
```

Comment:
`exceptionHandler` shouldn't be used in this case, if insights sending events fails, I believe we don't want it to report it to the UI